Content.Shared/Atmos/Rotting/AntiRottingOnBucleComponent.cs

```diff
@@ -0,0 +1,11 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Atmos.Rotting;
+
+/// <summary>
+/// Perishable entities buckled to an entity with this component this will not rot.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class ActiveAntiRottingOnBuckleComponent : Component
```

Comment:
this name is kinda wordy. Could easily just be called `AntiRotOnBuckleComponent`